### ISSUE TITLE
maximizing information entropy

### ISSUE THREAD

**Issue by ccran** [datawhalechina/joyrl#23]
https://github.com/datawhalechina/joyrl/blob/b36e8de522a663fb6cdfc34b2fa5d074d0fe9024/joyrl/algos/PPO/policy.py#L159C70-L159C102

Does maximizing information entropy mean that the sign in the loss function is negative?

`self.tot_loss = self.actor_loss + self.critic_loss - self.entropy_coef * entropy_mean`
